src/helpers/constants/ios.ts

```diff
@@ -1,13 +1,13 @@
 export const LOCAL_PATH_TO_CIO_NSE_FILES = `node_modules/customerio-expo-plugin/src/helpers/native-files/ios`;
 export const IOS_DEPLOYMENT_TARGET = '13.0';
-export const CIO_SDK_VERSION = '~> 2.0.0-alpha.1';
+export const CIO_SDK_VERSION = '2.0.0';
```

Comment:
```suggestion
export const CIO_SDK_VERSION = '~> 2.0.0';
```

Is there a reason the `~>` was removed?